senders/slack/slack.go

```diff
@@ -21,6 +22,12 @@ const (
 
 	messageMaxCharacters          = 40000
```

Comment:
Slack messages are actually limited to 4000 chars: https://api.slack.com/docs/rate-limits

Actually, it's 16 kilobytes for the entire JSON, including metadata. But 4000 chars is safe for any UTF-8 content.

Reply:
Will fix this once the algorithm for message building is finalized.